src/test/java/com/revature/studyforce/flashcard/service/RatingServiceTest.java

```diff
@@ -35,31 +36,50 @@
     @MockBean
     private RatingRepository ratingRepository;
     @MockBean
-    private FlashcardRepository flashcardRepo;
+    private FlashcardRepository flashcardRepository;
     @MockBean
     private UserRepository userRepository;
 
     @Autowired
     private RatingService ratingService;
 
     @Test
-    void whenCreateRating_returnsRatingResponseDTO(){
+    void givenRatingDTO_whenCreateRating_shouldReturnRatingResponseDTO(){
         List<Rating> rList = new ArrayList<>();
         User user = new User(0,"edson@revature.com","password","Edson","Rodriguez",true,false,false, Authority.USER, Timestamp.valueOf(LocalDateTime.now()),Timestamp.valueOf(LocalDateTime.now()));
```

Comment:
Using outdated user model, please correct to user model with 1 name field and no password field